client/components/RelatedItems/CardTemplate.jsx

```diff
@@ -1,19 +1,69 @@
 import React from 'react';
+import Rating from './Rating.jsx';
+import Comparison from './Comparison.jsx';
 
-const CardTemplate = (props) => {
-  return (
-    <li className="slide">
-      <div className="product-card">
-        <figure>
-          <img className="related-product-img" src={props.product.image} />
-          <div className="product-info">
-            <div>{props.product.name}</div>
-            <div>${Math.abs(Number(props.product.originalPrice))}</div>
+class CardTemplate extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      seen: false,
+      favorite: false
+    }
+    this.togglePop = this.togglePop.bind(this);
+    this.deselectOutfit = this.deselectOutfit.bind(this);
+    this.addToFavorite = this.addToFavorite.bind(this);
+  }
+
+  togglePop() {
+    this.setState({
+      seen: !this.state.seen
+    })
+  }
+
+  deselectOutfit() {
+    this.props.deselect();
+  }
```

Comment:
if all this function does is invoke a prop, you don't need the function. Invoke the prop directly on line 38 below